gravitee-apim-portal-webui-next/src/app/applications/applications.component.html

```diff
@@ -1,18 +1,40 @@
 <!--
 
     Copyright (C) 2024 The Gravitee team (http://gravitee.io)
-    
+
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-    
+
             http://www.apache.org/licenses/LICENSE-2.0
-    
+
     Unless required by applicable law or agreed to in writing, software
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
 
 -->
-<p>applications works!</p>
+
+@if (applicationPaginator$ | async; as applications) {
+  @if (applications.data.length === 0) {
+    <mat-card id="no-apps" appearance="outlined"> </mat-card>
+  } @else {
+    <mat-card>
+      <h2>Applications</h2>
+      <div class="padding-bottom">
+        An application represents a developer's project that interacts with the API. It acts as a means to manage access control to APIs via
+        subscriptions.
+      </div>
+      <div class="app-list__container" infiniteScroll (scrolled)="loadMoreApplications(applications)">
+        @for (application of applications.data; track application.id) {
+          <app-application-card [application]="application" />
+        }
+      </div>
+    </mat-card>
+  }
+}
+
+@if (loadingPage$ | async) {
+  <app-loader />
+}
```

Comment:
```suggestion
<mat-card appearance="outlined">
   <mat-card-content>
      <h2>Applications</h2>
      <div class="padding-bottom">
        An application represents a developer's project that interacts with the API. It acts as a means to manage access control to APIs via
        subscriptions.
      </div>
   @if (applicationPaginator$ | async; as applications) {
      @if (applications.data.length === 0) {
        
       } @else {
        <div class="app-list__container" infiniteScroll (scrolled)="loadMoreApplications(applications)">
          @for (application of applications.data; track application.id) {
            <app-application-card [application]="application" />
          }
        </div>
      }
    }
   
    @if (loadingPage$ | async) {
      <app-loader />
    }
  </mat-card-content>
</mat-card>

```

This puts everything in `mat-card-content`, so the padding is handled for you. This also adds `appearance="outlined"` to the root `mat-card`.

Reply:
Done